CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+# Changelog QualityOnDemand
+
+## Content
```

Comment:
```suggestion
## Table of Contents
```

Reply:
Done in [8ec7dba](https://github.com/camaraproject/QualityOnDemand/pull/99/commits/8ec7dba74e7a1a916a49c3439bfe4f328406fb7e)